Update technical-stack.md content (#299)
akeneo/pim-api-docs: content/guides/dam-connection/technical-stack.md

Both have their advantages and drawbacks.

For those who are not comfortable with these different ways to connect, below is a bit of literature to learn more about them.  

> For the others, you can skip this part. :wink:

Reviewer: ```suggestion
For the other ones, you can skip this part. :wink:
```

### API (REST/GraphQL)
It is the most common way to connect with an external application.

![API Schema](../../img/guides/dam_pim-connection_api.svg)

APIs are great for initialization purposes. But their main drawback lays in the fact you'll need a polling mechanism in order to monitor if something changed in the application.    

> In other words, you'll need to regularly ask what were the last updated DAM assets, in order to synchronize them with the PIM.

Reviewer: ```suggestion
In other words, you'll need to regularly ask which were the last updated DAM assets, in order to synchronize them with the PIM.
```

### Webhooks

> With webhooks, the DAM will send freshly created assets and updated assets values to your connector.

Reviewer: ```suggestion
With webhooks, the DAM will send newly created assets and updated asset values to your connector.
```

> You should avoid having too many calls to the PIM API and process only data you need. This is why the [DAM attribute selection](/pre-requisites.html#define-the-attributes-of-your-asset-families) is really important.

Reviewer: ```suggestion
You should avoid having too many calls to the PIM API and process only the data you need. This is why the [DAM attribute selection](/pre-requisites.html#define-the-attributes-of-your-asset-families) is really important.
```

Here are the [client errors](/documentation/responses.html#client-errors) that you can have on the PIM side, with our PHP client.

::: info

> If you are [creating/updating multiple assets at once](/php-client/exception.html), the API will return a list of errors for each attribute that it couldn't create or update successfully.  

Reviewer: ```suggestion
If you are [creating/updating multiple assets at once](/php-client/exception.html), the API will return a list of errors for each attribute that couldn't be created or updated successfully.  
```

You'll need to find a way to run the calls again for the assets that have not been synchronized.
:::

### Our skeleton

> As already stated in the [introduction](introduction.html#our-skeleton), we provide a skeleton from which you can take inspiration when writing your own connector.   

Reviewer: ```suggestion
As already stated in the [introduction](introduction.html#our-skeleton), we provide a skeleton to inspire you when writing your own connector.   
```
